pyproject.toml

```diff
@@ -1,8 +1,34 @@
 [build-system]
-requires = ['setuptools', 'setuptools-scm', 'wheel']
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
 
-[tool.setuptools_scm]
-version_scheme = "post-release"
+[project]
+name = "multiregex"
+authors = [
+  {name = "Jonas Haag", email = "jonas.haag@quantco.com"}
+]
+
+maintainers = [{name = "Jonas Haag", email = "jonas.haag@quantco.com"}]
```

Comment:
In the other projects we don't have `maintainers` but I don't have any opinions/knowledge about it